requirements.txt

```diff
@@ -12,7 +12,7 @@ blinker==1.4
 boto==2.49.0
 boutiques==0.5.25
 certifi==2018.11.29
-cffi==1.12.3
+cffi==1.15.0
```

Comment:
are those requirements associated to your PR? If not, maybe this could be a separate PR to update the requirements?